src/fsharp/FSharp.Core/string.fs

```diff
@@ -104,7 +107,5 @@ namespace Microsoft.FSharp.Core
 
         [<CompiledName("Length")>]
         let length (str:string) =
-            if String.IsNullOrEmpty str then
-                0
-            else
-                str.Length
+            if isNull str then 0
```

Comment:
This is unrelated change and might go into its own PR. Also I'd prefer the original indentation - or since it is now extremely short just put it on one line.

Reply:
@forki My bad, I made a mistake, it already has its own pr. I'll remove it here.